src/Algebra/Graph/ToGraph.hs

```diff
@@ -308,6 +328,14 @@ class ToGraph t where
     toAdjacencyMap :: Ord (ToVertex t) => t -> AM.AdjacencyMap (ToVertex t)
     toAdjacencyMap = foldg AM.empty AM.vertex AM.overlay AM.connect
 
+    -- | Convert a value to the corresponding 'LAM.LabelledAdjacencyMap'.
+    --
+    -- @
+    -- toAdjacencyMap == 'foldg' 'LAM.empty' 'LAM.vertex' 'LAM.overlay' 'LAM.connect'
+    -- @
+    toLabelledAdjacencyMap :: (Ord (ToVertex t), Dioid e )=> t -> LAM.LabelledAdjacencyMap (ToVertex t) e
+    toLabelledAdjacencyMap = foldg LAM.empty LAM.vertex LAM.overlay LAM.connect
```

Comment:
I have a feeling we'll actually need to add `class ToGraph t => ToLabelledGraph t` type class for this, with an additional associated type `ToLabel`. Let's also do this in a separate PR.